filament/src/materials/skybox.mat

```diff
@@ -116,11 +132,15 @@ fragment {
         } else if (materialParams.skyboxType == SHAPR_SKYBOX_TYPE_ENVIRONMENT) {
             sky = float4(textureLod(materialParams_skybox, variable_eyeDirection.xyz, 0.0).rgb, 1.0);
             sky.rgb *= frameUniforms.iblLuminance;
+        } else if (materialParams.skyboxType == SHAPR_SKYBOX_TYPE_CHECKERBOARD) {
```

Comment:
regarding branching, is materialParams.skyboxType a uniform value? do we need to think about optimization here?

Reply:
It is a uniform so at least the branch will be coherent. In terms of instruction counts, the best solution is to specialize the shader based on type and switch between those skybox shaders depending on what the user has selected.

Reply:
It is a uniform, and only changes when the type of environment is changed by the user. Later on we can create shader variants for each environment type, but for that, we need to make a deep dive into Filament's material compiler. This should be done with something akin to https://shapr3d.atlassian.net/wiki/spaces/DD/pages/2897641498/Visualization+Materials#Optimization-2 .

Reply:
Choosing shaders based on type sounds good. Is there a follow-up ticket for this on Jira?

Reply:
I think this should be after https://shapr3d.atlassian.net/browse/GFX-818 , and can be included in https://shapr3d.atlassian.net/browse/GFX-841

Reply:
sounds good!